eng/linkerTests/SupportFiles/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <PropertyGroup>
+    <PublishTrimmed>true</PublishTrimmed>
```

Comment:
PublishTrimmed default behaviour is not to link (unfortunately). You have to add 
```
	<_TrimmerDefaultAction>link</_TrimmerDefaultAction>
```

Reply:
Not sure I understand, I’m definitely seeing trimming happening.

Reply:
There are different trimming modes, the default one is the one not used by size sensitive workloads as it does only minimal trimming (never breaks stuff)

Reply:
I see interesting, I will use your property for now but I definitely echo @eerhardt sentiment that ideally we want to use documented public properties so that we are guaranteed that it won't get broken, specially in this case since if they decide to rename we would really never notice and we may start getting some tests passing incorrectly.

Reply:
Sure and the work is happening on that front as well